backend/src/main/java/com/it/p/lodz/pl/masi/configuration/SimpleCorsFilter.java

```diff
@@ -0,0 +1,42 @@
+package com.it.p.lodz.pl.masi.configuration;
+
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class SimpleCorsFilter implements Filter {
+
+    public SimpleCorsFilter() {
+    }
+
+    @Override
+    public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain) throws IOException, ServletException {
```

Comment:
Jeżeli używamy filtru to adnotacje do CORS w kontrolerach już chyba nie są potrzebne.